packages/contracts/test/token/erc20/governance-wrapped-erc20.ts

```diff
@@ -486,5 +495,419 @@ describe('GovernanceWrappedERC20', function () {
+                expect(await token.getVotes(to.address)).to.equal(200);
+              });
+              it('`to`s delegate has not changed', async () => {
+                expect(await token.delegates(to.address)).to.equal(toDelegate);
+              });
+              it('`to`s delegate has the correct voting power', async () => {
+                expect(await token.getVotes(toDelegate)).to.equal(200);
+              });
+
+              context('`to` transfers to `other`', async () => {
+                beforeEach(async () => {
+                  await expect(token.connect(to).transfer(other.address, 100))
+                    .to.emit(token, 'DelegateChanged')
+                    .withArgs(other.address, addressZero, other.address); // the transfer triggers automatic self-delegation for `other`
+                });
+
+                it('`to` has the correct voting power', async () => {
+                  expect(await token.getVotes(to.address)).to.equal(100); // 100 tokens are still left
+                });
+                it('`to`s delegate has not changed', async () => {
+                  expect(await token.delegates(to.address)).to.equal(
+                    toDelegate
+                  );
+                });
+                it('`to`s delegate has the correct voting power', async () => {
+                  expect(await token.getVotes(toDelegate)).to.equal(100);
+                });
+              });
+
+              context('`to` delegates to `other`', async () => {
+                beforeEach(async () => {
+                  await expect(token.connect(to).delegate(other.address))
+                    .to.emit(token, 'DelegateChanged')
+                    .withArgs(to.address, to.address, other.address); // `to` delegates to `other`
+                  toDelegate = other.address;
+                });
+
+                it('`to` has the correct voting power', async () => {
+                  expect(await token.getVotes(to.address)).to.equal(0);
+                });
+                it('`to`s delegate is correctly changed', async () => {
+                  expect(await token.delegates(to.address)).to.equal(
+                    toDelegate
+                  );
+                });
+                it('`to`s delegate has the correct voting power', async () => {
+                  expect(await token.getVotes(toDelegate)).to.equal(200);
+                });
+              });
+            }
+          );
+
+          context('`to` receives via transfer from `from`', async () => {
+            beforeEach(async () => {
+              await token.depositFor(from.address, 100);
+              await expect(
+                token.connect(from).transfer(to.address, 100)
+              ).to.not.emit(token, 'DelegateChanged');
+            });
+
+            it('`from` has the correct voting power', async () => {
+              expect(await token.getVotes(to.address)).to.equal(200);
```

Comment:
check doesn't match the description

Reply:
Fixed in c0d3b11a35f333f3c0dac00e73c4f7f2edb89620